src/components/Confirm/index.js

```diff
@@ -9,30 +8,35 @@ import { preventDefaultOnClick } from '~utils/index'
 import { uploadDocument, uploadLivePhoto, uploadLiveVideo } from '~utils/onfidoApi'
 import CaptureViewer from './CaptureViewer'
 import { poaDocumentTypes } from '../DocumentSelector/documentTypes'
+import Button from '../Button'
 import Error from '../Error'
 import Spinner from '../Spinner'
 import Title from '../Title'
 import { trackException, trackComponentAndMode, appendToTracking, sendEvent } from '../../Tracker'
 import { localised } from '../../locales'
 
 const RetakeAction = localised(({retakeAction, translate}) =>
-  <button onClick={retakeAction}
-    className={`${theme.btn} ${theme['btn-outline']} ${style.retake}`}>
+  <Button
+    onClick={retakeAction}
+    className={style.retake}
+    variants={["outline"]}
+  >
     {translate('confirm.redo')}
-  </button>
+  </Button>
 )
 
 const ConfirmAction = localised(({confirmAction, translate, error}) =>
-  <button href='#' className={`${theme.btn} ${theme["btn-primary"]}`}
+  <Button
+    className={style["btn-primary"]}
+    variants={["primary"]}
```

Comment:
I can see that the style of this class is overriding the style in `style.button-primary`.

Reply:
yes, that's the idea. what is your concern?

Reply:
The button on small screens should have [this width](https://github.com/onfido/onfido-sdk-ui/blob/4eb4616d59ab86d1184c8603dc3a013d346eb0fe/src/components/Confirm/style.css#L43) instead of [this one](https://github.com/onfido/onfido-sdk-ui/blob/4eb4616d59ab86d1184c8603dc3a013d346eb0fe/src/components/Button/style.css#L40). 
<img width="1440" alt="screen shot 2019-02-18 at 12 08 57" src="https://user-images.githubusercontent.com/7127427/52950234-67138200-3376-11e9-9b87-484f56f4c506.png">

Reply:
got it now! I actually thought you meant the opposite. On my side I didn't see the bug, but I tested it on webpack dev server, which probably reorders the styles. Nonetheless, they both have the same specificity, so it needs to be fixed! Good catch.

fix here: https://github.com/onfido/onfido-sdk-ui/pull/552/commits/26871d33f0135f156c8841097c7f81ed3bb81cc5

Reply:
btw, do you know why `button-primary` has 
```
@media (--small-viewport) {
    width: 100%;
  }
``` ? It feels like a layout logic, probably better placed outside of the button.

Reply:
On small screens when there is only one action, the button takes 100% of the width (see Welcome screen and video intro screen)

Reply:
I see. I wonder whether we should move that logic outside of the button at some point then. Feels like a job of the parent to decide if the button takes the entire width.

Reply:
Yes, I agree. Not ideal to have this logic in the theme.

Reply:
it's not in the theme. it's in the Button/style.css